aerogear/mobile-security-service
# feat(icon): Added Aerogear icon as favicon (AEROGEAR-8884)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8884

## What

Replaced default React favicon with the Aerogear icon.

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Open the application in your browser.
2. Clear your browser cache (press Cmd+F5 or Ctrl+F5 a few times).
3. Does the icon in the browser tab change?

## Checklist:


## Progress


## Additional Notes

![Screenshot from 2019-03-22 16-26-04](https://user-images.githubusercontent.com/11743717/54837821-3bf6b800-4cbf-11e9-8ee0-d1395f442fbe.png)
